frontend/src/metabase/query_builder/components/view/ViewHeader/components/QuestionActions/QuestionMoreActionsMenu/QuestionMoreActionsMenu.tsx

```diff
@@ -162,8 +177,18 @@ export const QuestionMoreActionsMenu = ({
         {t`Edit settings`}
       </Menu.Item>
     ),
+    canUseEmbeddingActions && (
+      <EmbeddingQuestionActions
+        key="embeddingActions"
```

Comment:
nit: use `dash-case` for consistency with other items below

```suggestion
        key="embedding-actions"
```